src/locomoset/metrics/run.py

```diff
@@ -104,6 +104,29 @@ def compute_metric(config: dict) -> dict:
     return results
 
 
+def compute_task_agnostic_metric(config: dict) -> dict:
```

Comment:
We may need to revisit this and have a structure that makes it easier to abstract away different types of metrics, but as long as we only have two choices this works fine 👍